scanner/keppel/main.go

```diff
@@ -66,121 +67,134 @@ func main() {
 	if err != nil {
 		log.WithError(err).Fatal("Error during scanner setup")
 	}
-	accounts, err := keppelScanner.ListAccounts()
+
+	// Get components
+	components, err := keppelProcessor.GetAllComponents(nil, 100)
```

Comment:
Why don't we make a GraphQL query that fetches components and versions at once?

Reply:
🙈 Of, course! Excellent idea.

Reply:
Implemented.